src/styles/PageHero.scss

```diff
@@ -0,0 +1,22 @@
+.page-hero{
+	width: 100%;
+    height: 100vh;
+    overflow: hidden;
+    position: relative;
+
+	&_img{
+	    width: 100%;
+	    height: 100%;
+	    object-fit: cover;
+	    object-position: center;
+	    position: fixed;
+	}
+
+	&_container{
+		position: absolute;
+    	bottom: 2em;
+    	color: $white;
+    	left: 50%;
+    	transform: translateX(-50%);
+	}
```

Comment:
Just a slight adjustment to indentation here. We should sit down and figure out if we can find a way to make it easier to maintain good indentation. Mostly for readability.